PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1351,46 +1351,29 @@ void Notepad_plus::removeEmptyLine(bool isBlankContained)
 	_findReplaceDlg.processAll(ProcessReplaceAll, &env, true);
 }
 
-void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir)
+void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir, const Progress* progress)

```

Comment:
To be fair about my own pedantry, using `const` in places where it isn't *strictly necessary* looks better and more readable to someone (like me) who is a self-described *insufferable `const` nazi*.